docs/content/docs/operations/configuration.md

```diff
@@ -22,8 +22,261 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-{{< hint warning >}}
-**TODO**: How to configure Flink Agents for local execution and Flink cluster execution.
+## How to configure Flink Agents
 
-**TODO**: List of all built-in Configuration options.
-{{< /hint >}}
\ No newline at end of file
+There are three ways to configure Flink Agents:
+
+1. **Explicit Settings in Code**
+2. **YAML Configuration File**
+3. **Configuration During Build Agent**
```

Comment:
Both 1) and 3) are in codes. I would suggest to describe these as:
1. Setting via the AgentsExecutionEnvironment
2. Setting via a YAML configuration file
3. Setting via the ResourceDescriptor

Reply:
And we should explain the scope